src/main/java/com/github/manolo8/darkbot/gui/titlebar/ExtraButton.java

```diff
@@ -31,12 +32,44 @@
     private static final Set<ExtraMenuProvider> EXTRA_DECORATIONS = new LinkedHashSet<>();
     private static final Lazy<Void> clean = new Lazy.NoCache<>();
 
-    public static void setExtraDecorations(Stream<ExtraMenuProvider> provider) {
+    public static void setExtraDecorations(Stream<ExtraMenuProvider> provider, FeatureRegistry featureRegistry) {
         EXTRA_DECORATIONS.clear();
-        provider.forEach(EXTRA_DECORATIONS::add);
+        PluginExtraMenuProvider.plugins.clear();
+
+        provider.forEach(extra -> {
+            if (!(extra instanceof PluginExtraMenuProvider) && !(extra instanceof DefaultExtraMenuProvider)) {
+                Plugin plugin = featureRegistry.getFeatureDefinition(extra).getPlugin();
```

Comment:
```suggestion
            Plugin pl = featureRegistry.getFeatureDefinition(extra).getPlugin();
            if (pl != null) {
```
If the plugin is null, it means they're native features